bears/rest/RSTcheckBear.py

```diff
@@ -0,0 +1,58 @@
+
+import re
+
+from rstcheck import check
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class RSTcheckBear(LocalBear):
+    LANGUAGES = {"reStructuredText"}
+    REQUIREMENTS = {PipRequirement('rstcheck', '2.*')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/ccjtkotaqzgggkkqtgelmj02x'
+
+    def run(self, filename, file,
+            code_block_language_ignore: list):
```

Comment:
a default value for code_block_language_ignore? Doesn't coala bypass asking for input if defaults have been set?